### PR TITLE
Medium refactor

### PR DESCRIPTION
**Pull request by benrudolph**
@dannyroberts doing a couple passes over the next week or so to break apart our 1400 line fabfile. was playing at making resumable deploys over the weekend and realized the code has just got to big to reason about easily

cc: @biyeun

### CODE REVIEW

File: fab/operations/db.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import datetime
```
Comment (benrudolph) [inline fab/operations/db.py:1]: going to start splitting out major operations like `db`, `services`, `staticfiles` etc into the `operations` module. everything in an operation module should not depend on the global state `env` and it should not define any `@task`s. at least that's the goal thus far

**Comment by benrudolph** [comment]
put tag on to allow for some time to test on staging

File: fab/fabfile.py [hunk 2]
```diff
@@ -1034,32 +1032,6 @@ def flip_es_aliases():
         sudo('%(virtualenv_root)s/bin/python manage.py ptop_es_manage --flip_all_aliases' % env)
 
 
-@roles(ROLES_DJANGO)
-@parallel
-def update_manifest(save=False, soft=False, use_current_release=False):
```
Comment (benrudolph) [inline fab/fabfile.py:1117]: this hasn't been necessary since we switched to atomic deploys

**Comment by dannyroberts** [comment]
I am very pro this concept. Let me know when you've had a chance to test a bit / you want me to take a closer look and merge

**Comment by benrudolph** [comment]
closing in favor of: https://github.com/dimagi/commcare-hq-deploy/pull/53

**Pull request closed without merging**